ebean-migration/src/test/java/io/ebean/migration/MigrationTableAsyncTest.java

```diff
@@ -151,6 +170,21 @@ private void runTest() throws SQLException, InterruptedException, ExecutionExcep
     dropTable("m1");
     dropTable("m2");
     dropTable("m3");
+    
+    if (withExisting) {
+      // create empty migration table
```

Comment:
I added tests where it checks, if the migration will also work (so, if table exists, but contains no run migrations, yet)